### ISSUE TITLE
use `env!("CARGO_PKG_VERSION")` for version

### ISSUE THREAD

**Issue by vitiral** [casey/just#36]
You no longer need to manually update your version! Super nice :)
```
.version(env!("CARGO_PKG_VERSION"))
```

**Comment by kbknapp** [comment]
clap supports this by using the `crate_version!` macro.

A common thing to do is:

``` rust
.version(concat!("v", crate_version!())
```

**Comment by vitiral** [comment]
Hey, that looks even better, I'll have to use it

**Comment by vitiral** [comment]
Another nice thing is that this makes your publish command simpler

**Comment by casey** [comment]
Awesome, all great suggestions. Thanks a bunch! Implemented and merged in PR #37.

I elected to go with:

`.version(concat!("v", env!("CARGO_PKG_VERSION"))`

In order to bring the nicer looking `crate_version!` into scope, I would need a `#[macro_use]` above `extern crate clap` in `app.rs`.

However, `#[macro_use]` is only allowed in the crate root, which is `lib.rs` instead of `app.rs`. I would rather keep the crate declaration in `app.rs`, since that's the only place it's used, thus why I went with the less attractive version.
